keras_preprocessing/text.py

```diff
@@ -268,13 +271,11 @@ def texts_to_sequences(self, texts):
         # Returns
             A list of sequences.
         """
-        res = []
-        for vect in self.texts_to_sequences_generator(texts):
-            res.append(vect)
+        res = list(self.texts_to_sequences_generator(texts))
```

Comment:
why not just return? Why create a variable?

Reply:
Done